folium/vector_layers.py

```diff
@@ -287,7 +287,7 @@ class CircleMarker(Marker):
         Radius of the circle marker, in pixels.
     **kwargs
         Other valid (possibly inherited) options. See:
-        https://leafletjs.com/reference-1.4.0.html#circlemarker
+        https://leafletjs.com/reference-1.5.1.html#circlemarker
```

Comment:
@Conengmo looks like this URL re-directs to 1.5.0 and makes me think that they are not going to publish a 1.5.1 version of the docs. I guess that we should make the sphinx link check a soft fail so we can merge this and create our docs. What do you think?

Reply:
You're right, it seems they always only publish docs for the major and minor versions. I don't know that much about Sphinx. I'm thinking now it may be easiest to only link to their documentation of the minor versions as well? So change all documentation links to 'reference-1.5.0'. Those links will also perhaps resolve faster since there's no redirect.

Reply:
It is properly redirected so I just moved the `linkcheck` part of sphinx to a "allow failures" tests. It was meant to be just a report and not really a hard error.